packages/manager/apps/dedicated/client/app/account/account.routing.js

```diff
@@ -1,7 +1,7 @@
 export default /* @ngInject */ ($stateProvider) => {
   [
     {
-      abstract: true,
+      abstract: false,
```

Comment:
**question:** there are no url, template or component associated with that state. Are you sure about removing the abstract flag ? What is displayed when you redirect to that state ?